lib/osf-components/addon/components/tags-widget/component.ts

```diff
@@ -22,6 +23,7 @@ export default class TagsWidget extends Component.extend({ styles }) {
 
     // required arguments
     taggable!: Taggable;
+    isPreprint = false;
```

Comment:
minor nit: since this isn't a required arg, let's move this to the `// optional arguments` section below. Also, we do we want to rename this something a bit more generic/descriptive like `tagClickable` or something like that?